testing/time_test.ts

```diff
@@ -69,62 +69,21 @@ Deno.test("Fake Date instance methods passthrough to real Date instance methods"
   try {
     const now = new Date("2020-05-25T05:00:00.12345Z");
     assertEquals(now.toISOString(), "2020-05-25T05:00:00.123Z");
-    Object.getOwnPropertyNames(_internals.Date.prototype).forEach(
-      (method: string) => {
-        if (
-          typeof _internals.Date.prototype[method as keyof Date] === "function"
-        ) {
-          if (typeof now[method as keyof Date] !== "function") {
-            throw new MockError(`FakeDate missing ${method} method`);
-          }
-          const returned = Symbol();
-          const func = stub(
-            _internals.Date.prototype,
-            method as keyof Date,
-            () => returned,
-          );
-          try {
-            const args = Array(5).fill(undefined).map(() => Symbol());
-            (now[method as keyof Date] as CallableFunction)(...args);
-            assertSpyCall(func, 0, {
-              args,
-              returned,
-            });
-            assertInstanceOf(func.calls[0].self, _internals.Date);
-          } finally {
-            func.restore();
-          }
-        }
-      },
-    );
-    Object.getOwnPropertySymbols(_internals.Date.prototype).forEach(
-      (method: symbol) => {
-        if (
-          typeof _internals.Date.prototype[method as keyof Date] === "function"
-        ) {
-          if (typeof now[method as keyof Date] !== "function") {
-            throw new MockError(`FakeDate missing ${method.toString()} method`);
-          }
-          const returned = Symbol();
-          const func = stub(
-            _internals.Date.prototype,
-            method as keyof Date,
-            () => returned,
-          );
-          try {
-            const args = Array(5).fill(undefined).map(() => Symbol());
-            (now[method as keyof Date] as CallableFunction)(...args);
-            assertSpyCall(func, 0, {
-              args,
-              returned,
-            });
-            assertInstanceOf(func.calls[0].self, _internals.Date);
-          } finally {
-            func.restore();
-          }
-        }
-      },
+
+    const func = spy(
+      _internals.Date.prototype,
+      "toISOString",
     );
+    try {
+      now.toISOString();
+      assertSpyCall(func, 0, {
```

Comment:
For this test, I think it would also be good to test that a symbol calls through to the original symbol too.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/@@toPrimitive

For that, you would spy on Symbol.toPrimitive as the property. Then if you do `${now}` it should call it with 'string' as the arg and return a string.